docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+##### enable_materialized_view
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to enable the creation of materialized views.
+
+##### enable_decimal_v3
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to support the DECIMAL V3 data type.
+
+##### enable_sql_blacklist
+
+- Unit: -
+- Default: FALSE
+- Description: Whether to enable blacklist check for SQL queries. When this feature is enabled, queries in the blacklist cannot be executed.
+
+##### dynamic_partition_check_interval_seconds
+
+- Unit: s
+- Default: 600
+- Description: The interval at which new data is checked. If new data is detected, StarRocks automatically creates partitions for the data.
+
+##### dynamic_partition_enable
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to enable the dynamic partitioning feature. When this feature is enabled, StarRocks dynamically creates partitions for new data and automatically deletes expired partitions to ensure the freshness of data.
+
+##### max_partitions_in_one_batch
+
+- Unit: -
+- Default: 4096
+- Description: The maximum number of partitions that can be created when you bulk create partitions.
+
+##### max_query_retry_time
+
+- Unit: -
+- Default: 2
+- Description: The maximum number of query retries on an FE.
+
+##### max_create_table_timeout_second
+
+- Unit: s
+- Default: 600
+- Description: The maximum timeout duration for creating a table, in seconds.
+
+##### create_table_max_serial_replicas
+
+- Unit: -
+- Default: 128
+- Description: The maximum number of replicas to create serially. If actual replica count exceeds this, replicas will be created concurrently. Try to reduce this config if table creation is taking a long time to complete.
+
+##### max_running_rollup_job_num_per_table
+
+- Unit: -
+- Default: 1
+- Description: The maximum number of rollup jobs can run in parallel for a table.
+
+##### max_planner_scalar_rewrite_num
+
+- Unit: -
+- Default: 100000
+- Description: The maximum number of times that the optimizer can rewrite a scalar operator.
+
+##### enable_statistic_collect
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to collect statistics for the CBO. This feature is enabled by default.
+
+##### enable_collect_full_statistic
+
+- Unit: -
+- Default: TRUE
+- Description: Whether to enable automatic full statistics collection. This feature is enabled by default.
+
+##### statistic_auto_collect_ratio
+
+- Unit: -
+- Default: 0.8
+- Description: The threshold for determining whether the statistics for automatic collection are healthy. If statistics health is below this threshold, automatic collection is triggered.
+
+##### statistic_max_full_collect_data_size
+
+- Unit: LONG
+- Default: 107374182400
+- Description: The size of the largest partition for automatic collection to collect data. Unit: Byte. If a partition exceeds this value, full collection is discarded and sampled collection is performed instead.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/908943d2-2ea5-4bcc-974c-eafa83318a9b)
1. Which unit is correct?
2. Remove the unit in "Description".

Reply:
yes, this is confusing.  The unit is a Long, but it represents the number of Bytes.  I will reword the sentence.